backend/api/submissions/permissions.py

```diff
@@ -3,6 +3,7 @@
 from api.permissions import HasTokenPermission
 from submissions.models import Submission
 from voting.helpers import pastaporto_user_info_can_vote
+from voting.models.ranking import RankRequest
 
 
 class CanSeeSubmissionRestrictedFields(BasePermission):
```

Comment:
not for this PR but something I think we should check if we have time: How do we handle cancelled submissions permission wise? what happens if I query a cancelled submission directly by doing `submission(id: XX)`? Ideally it should fail with a 404

Reply:
Good point

Reply:
yeah they are visible!

Reply:
We should change that only the Author or an admin can see!